### DISCUSSION TITLE
提交问题后得不到有效的反馈

### DISCUSSION THREAD

**Discussion by xc-bnu** [getlantern/lantern-forum-cn#1103] [General]
lantern自从今年5月份左右购买以来，一直都存在连不上外网的情况（如下图），网速非常慢。
![6cb3bc041d2690ef923fb2026ae793f](https://github.com/user-attachments/assets/af3f1a00-9dfb-45c9-ae2d-aa664d14968d)
一直都有提交我这边存在的问题，然后一直得到的反馈是：
![4658674c5fdad53b0d6f2e703febc1c](https://github.com/user-attachments/assets/9f03dd78-1097-422c-830e-9d0520a64087)
得到反馈后，问题一直没得到解决。最后一句话实在是让我不解，致力于大客户？什么样的才是大客户？就现在你们这解决问题的能力，能致力于大客户吗？

**Comment by xc-bnu** [comment]
@Mike777Khoo 我之前也跟你反馈过网速慢，连不上网的情况，一直得不到解决

**Reply by MaxMKMax** [reply]
@xc-bnu 您好. 这是美国团队用谷歌翻译照成的误会. 这个翻译错了吧. 他们也不清楚中文. 请您原谅. 谢谢!

**Comment by MaxMKMax** [comment]
@xc-bnu 您好. 这是美国团队用谷歌翻译照成的误会. 这个翻译错了吧. 他们也不清楚中文. 请您原谅. 谢谢!

对了. 请问您有更新到最新版本吗? 您帮我提交个报告吧, 方便我和他们沟通. 谢谢. 

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
